survey-sdk/src/main/java/co/nimblehq/survey/sdk/SurveyApi.kt

```diff
@@ -24,6 +26,21 @@ class SurveyApi private constructor(): NetworkBuilder() {
         return this
     }
 
+    fun withClientId(clientId: String): SurveyApi {
+        this.clientId = clientId
+        return this
+    }
+
+    fun withClientSecret(clientSecret: String): SurveyApi {
+        this.clientSecret = clientSecret
+        return this
+    }
```

Comment:
The `ClientId` and `ClientSecret` are quired added to the SDK as the Credentials 🤔  What do you think about creating `Credentials(clientId, clientSecret)` for usage?

Reply:
@hoangnguyen92dn I think it a good idea, let me update it.

Reply:
@doannimble I mean that create an object called `Credentials(clientId, clientSecret)` 😄  The same as Login request with usernam and password.

Reply:
@hoangnguyen92dn No worry about it. I've update the way to put the credentials 🎸